app/lib/common/providers/common_providers.dart

```diff
@@ -135,3 +135,5 @@ final canRedactProvider = FutureProvider.autoDispose.family<bool, dynamic>(
     }
   }),
 );
+
+final searchValueProvider = StateProvider.autoDispose<String>((ref) => '');
```

Comment:
Added `searchValueProvider` in common providers file as it is common provider used across different use-cases.